generator/feature_sorter.cpp

```diff
@@ -281,15 +289,10 @@ class FeaturesCollector2 : public FeaturesCollector
   static void SimplifyPoints(int level, bool isCoast, m2::RectD const & rect, Points const & in, Points & out)
   {
     if (isCoast)
-    {
-      DistanceToSegmentWithRectBounds fn(rect);
-      feature::SimplifyPoints(fn, level, in, out);
-    }
+      feature::SimplifyPoints(DistanceToSegmentWithRectBounds(rect), level, in, out);
     else
-    {
       feature::SimplifyPoints(m2::SquaredDistanceFromSegmentToPoint(), level, in, out);
     }
-  }
```

Comment:
wrong indent left

Reply:
The purpose is to keep readable diff in this not obvious block.

Reply:
the diff is bogus also, that's how I did notice this mistake :)
and the result is obviously very bad readability and comprehension-wise:
![image](https://github.com/organicmaps/organicmaps/assets/18434508/7f7e3d1f-0c86-4e68-9b75-b4ea52e1766e)